### DISCUSSION TITLE
Setting conditional word wrap and split views

### DISCUSSION THREAD

**Discussion by daron81** [orbitalquark/textadept#490] [Q&A]
Hello,

I've had some unexpected behavior with word wrap when splitting views.

I have the following config in my init.lua to only enable wrapping for certain file types:
`
events.connect(events.LEXER_LOADED, function(name)
  if name == 'markdown' or name == 'text' then
    view.wrap_mode = view.WRAP_WHITESPACE
  else
    view.wrap_mode = view.WRAP_NONE
  end
end)
`

However, when I split a view with that opens one of those file types, the new view does not have wrapping.

Is there an event that will consistently apply the wrapping to these file types in all cases, or will I have to add this to every view event as well? Thanks in advance for any help!

**Comment by orbitalquark** [comment]
Did you try connecting to `events.VIEW_NEW`?
